File: UI/eturistickaagencija_mobile/lib/model/destinacija.dart
import 'package:json_annotation/json_annotation.dart';

part 'destinacija.g.dart';

@JsonSerializable()
class Destinacija{
int? id;
String? naziv;
String? slika;
int? gradId;

Destinacija(this.id,this.naziv,this.slika,this.gradId);

factory Destinacija.fromJson(Map<String,dynamic>json)=>_$DestinacijaFromJson(json);
Map<String,dynamic>toJson()=>_$DestinacijaToJson(this);
}

File: eTuristickaAgencija.Models/Destinacija.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTuristickaAgencija.Models
{
    public class Destinacija
    {
        public int Id { get; set; }
        public byte[] Slika { get; set; }
        public string Naziv { get; set; }
        public int GradId { get; set; }
        public string StateMachine { get; set; }
    }
}
